pontoon/settings/base.py

```diff
@@ -736,6 +739,9 @@ def _allowed_hosts():
 
 SYNC_LOG_RETENTION = 90  # days
 
+SYNC_VCS_NAME = os.environ.get("SYNC_VCS_NAME", "Pontoon")
+SYNC_VCS_EMAIL = os.environ.get("SYNC_VCS_EMAIL", "pontoon@example.com")
```

Comment:
I'd add call these `VCS_SYNC_X` and move them up, right under `LOCALE_REQUEST_EMAIL` (to keep account/email settings together).

I'd also add a comment on top, e.g.:
```
VCS identity to be used when committing translations.
```